docs/rancher-setup.md

```diff
@@ -1,17 +1,64 @@
 # Rancher Setup
 
-## Setup instructions modification
-When using Rancher, the recommended certificate option is to use Rancher generated certificates. However, there is a modification to the step in the provided [instructions](https://rancher.com/docs/rancher/v2.x/en/installation/k8s-install/helm-rancher/). 
+## Add the Helm Chart Repository
+Use helm repo add command to add the Helm chart repository that contains charts to install Rancher. For more information about the repository choices and which is best for your use case, see Choosing a Version of Rancher.
```

Comment:
'Use *the* `helm repo add` command'? (And suggest marking helm repo add as code).

Should 'Choosing a Version of Rancher' be a link?

Reply:
Updated